dev-guide/cluster-fleet-evaluation.md

```diff
@@ -0,0 +1,160 @@
+feedback decide on the level of investment required to properly inform the
+customer base, or change implementation details.
+
+## Solution
+
+Rather than introducing the change, add just the detection logic to the operator
+for the change and associate it with a
+[ClusterStatusConditionTypes](https://github.com/openshift/api/blob/cc0db1116639638254e87a564902833f1ee006d5/config/v1/types_cluster_operator.go#L145)
+called _ClusterFleetEvaluation_. If the
+[ConditionStatus](https://github.com/openshift/api/blob/cc0db1116639638254e87a564902833f1ee006d5/config/v1/types_cluster_operator.go#L124)/
+[Reason](https://github.com/openshift/api/blob/cc0db1116639638254e87a564902833f1ee006d5/config/v1/types_cluster_operator.go#L133)
+associated with this type is
+[ConditionTrue](https://github.com/openshift/api/blob/cc0db1116639638254e87a564902833f1ee006d5/config/v1/types_cluster_operator.go#L107)
+or matches a new _ConditionStatus/Reason_ you are expecting, it will indicate
+that the change would have affected that cluster. You can join multiple
+_ConditionStatus/Reason_ to test multiple scenarios. Now release the detection
+logic to master and
+[backport](https://docs.google.com/document/d/1PC87sSFa_zGCk95kXDW-wrVxnlgBmkHqpOgQnd4bbUw/edit)
+it to a N-1 Z stream release where N is the current release under development.
+The Z stream backport is useful as it allows the team to gather the data without
+going through a full release cycle. The backport should follow the
+[OpenShift backport process](https://docs.google.com/document/d/1PC87sSFa_zGCk95kXDW-wrVxnlgBmkHqpOgQnd4bbUw/edit).
+How far back the backports should be done is subjective and the teams should aim
+to backport as far back as possible so as to collect a reasonable amount data
+that allows them to reach a conclusion.
+
+Once the Z streams have been released the component developers can [interrogate
+the telemetry data](#interacting-with-telemetry) to figure out the prevalence of
+the scenario that the change will introduce. Allow sufficient time (TBD) for
+customers to upgrade to the Z stream release. The upgrade data is also available
+in telemetry.
+
+At this point the team needs to validate that the telemetry data matches their
+expectations and make a decision if they still plan to move forward with the
+change. In particular the team needs to measure the level of impact the change
+has. If the percentage of impact is very low then it might be preferable to
+contact the affected customers. Alternatively if the percentage of impact is
+very high, it would be preferable that the team re-evaluate the change and
+investigate if there is a way to automate the fix process in a seamless manner.
+
+If the component team decides to move forward, they will then be responsible for
+writing a [Knowledge Centered Support](https://source.redhat.com/groups/public/cee-kcs-program/cee_kcs_program_wiki/kcs_solutions_content_standard_v20)
```

Comment:
nit: another internal doc link, which might benefit from `(internal link)` or some other marker to reduce external click-through frustration.